lib/money-rails.rb

```diff
@@ -5,6 +5,7 @@
 require "money-rails/money"
 require "money-rails/version"
 require 'money-rails/hooks'
+require 'money-rails/errors'
```

Comment:
What about `money-rails/error` instead and using singular form for the filename `lib/money-rails/error.rb`?

I think it could be more coherent with `MoneyRails::Error`, no?